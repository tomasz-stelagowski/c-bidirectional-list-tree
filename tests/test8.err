NODES: 2
NODES: 3
NODES: 4
NODES: 5
NODES: 6
NODES: 7
NODES: 8
NODES: 9
NODES: 10
NODES: 11
NODES: 12
NODES: 13
NODES: 14
NODES: 15
NODES: 16
NODES: 17
NODES: 18
NODES: 19
NODES: 20
NODES: 21
NODES: 22
NODES: 23
NODES: 24
NODES: 25
NODES: 26
NODES: 27
NODES: 28
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 27
NODES: 28
NODES: 29
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 28
NODES: 29
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 26
NODES: 27
NODES: 28
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 12
NODES: 13
NODES: 14
NODES: 15
NODES: 16
NODES: 17
NODES: 18
NODES: 19
NODES: 20
NODES: 21
NODES: 22
NODES: 23
NODES: 24
NODES: 25
NODES: 26
NODES: 27
NODES: 28
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 26
NODES: 27
NODES: 28
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 25
NODES: 26
NODES: 27
NODES: 28
NODES: 29
NODES: 30
NODES: 31
NODES: 28
NODES: 29
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 19
NODES: 20
NODES: 21
NODES: 22
NODES: 23
NODES: 24
NODES: 25
NODES: 26
NODES: 27
NODES: 28
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 29
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 29
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 31
NODES: 30
NODES: 31
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 30
NODES: 30
