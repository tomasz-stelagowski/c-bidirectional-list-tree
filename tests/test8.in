ADD_NODE 0
SPLIT_NODE 0 1
ADD_NODE 1
SPLIT_NODE 0 1
ADD_NODE 1
SPLIT_NODE 1 3
ADD_NODE 4
SPLIT_NODE 4 2
ADD_NODE 3
SPLIT_NODE 1 6
ADD_NODE 8
SPLIT_NODE 8 7
ADD_NODE 3
SPLIT_NODE 8 7
ADD_NODE 12
SPLIT_NODE 4 2
ADD_NODE 6
SPLIT_NODE 4 2
ADD_NODE 10
SPLIT_NODE 14 12
ADD_NODE 13
SPLIT_NODE 14 12
ADD_NODE 15
SPLIT_NODE 8 14
ADD_NODE 12
SPLIT_NODE 14 22
ADD_NODE 22
SPLIT_NODE 8 7
ADD_NODE 20
DELETE_NODE 11
ADD_NODE 2
DELETE_SUBTREE 29
ADD_NODE 2
RIGHTMOST_CHILD 26
RIGHTMOST_CHILD 17
ADD_NODE 31
DELETE_NODE 9
SPLIT_NODE 14 12
DELETE_SUBTREE 2
SPLIT_NODE 1 10
ADD_NODE 14
SPLIT_NODE 10 19
ADD_NODE 33
DELETE_NODE 12
SPLIT_NODE 33 22
DELETE_SUBTREE 19
RIGHTMOST_CHILD 33
RIGHTMOST_CHILD 26
RIGHTMOST_CHILD 20
ADD_NODE 33
DELETE_NODE 34
SPLIT_NODE 6 17
DELETE_SUBTREE 24
RIGHTMOST_CHILD 18
RIGHTMOST_CHILD 39
RIGHTMOST_CHILD 38
ADD_NODE 13
DELETE_NODE 22
SPLIT_NODE 14 15
DELETE_SUBTREE 21
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 17
RIGHTMOST_CHILD 5
ADD_NODE 18
DELETE_NODE 20
SPLIT_NODE 42 25
DELETE_SUBTREE 23
RIGHTMOST_CHILD 5
RIGHTMOST_CHILD 42
RIGHTMOST_CHILD 35
ADD_NODE 42
DELETE_NODE 15
SPLIT_NODE 6 40
DELETE_SUBTREE 45
RIGHTMOST_CHILD 4
RIGHTMOST_CHILD 28
RIGHTMOST_CHILD 14
ADD_NODE 18
DELETE_NODE 40
SPLIT_NODE 44 33
DELETE_SUBTREE 46
RIGHTMOST_CHILD 16
RIGHTMOST_CHILD 28
RIGHTMOST_CHILD 17
ADD_NODE 39
DELETE_NODE 3
SPLIT_NODE 33 39
DELETE_SUBTREE 50
RIGHTMOST_CHILD 4
RIGHTMOST_CHILD 17
RIGHTMOST_CHILD 41
ADD_NODE 48
DELETE_NODE 13
SPLIT_NODE 8 28
DELETE_SUBTREE 52
RIGHTMOST_CHILD 49
RIGHTMOST_CHILD 39
RIGHTMOST_CHILD 1
ADD_NODE 33
DELETE_NODE 43
SPLIT_NODE 0 4
DELETE_SUBTREE 48
ADD_NODE 18
SPLIT_NODE 44 33
RIGHTMOST_CHILD 6
ADD_NODE 16
DELETE_NODE 41
SPLIT_NODE 18 55
DELETE_SUBTREE 49
RIGHTMOST_CHILD 39
RIGHTMOST_CHILD 33
RIGHTMOST_CHILD 14
ADD_NODE 6
DELETE_NODE 4
SPLIT_NODE 33 39
DELETE_SUBTREE 55
RIGHTMOST_CHILD 5
RIGHTMOST_CHILD 37
RIGHTMOST_CHILD 8
ADD_NODE 28
DELETE_NODE 25
SPLIT_NODE 14 42
DELETE_SUBTREE 58
RIGHTMOST_CHILD 33
RIGHTMOST_CHILD 8
RIGHTMOST_CHILD 42
ADD_NODE 10
DELETE_NODE 6
SPLIT_NODE 10 63
DELETE_SUBTREE 26
RIGHTMOST_CHILD 47
RIGHTMOST_CHILD 27
RIGHTMOST_CHILD 28
ADD_NODE 37
DELETE_NODE 57
SPLIT_NODE 10 36
DELETE_SUBTREE 10
SPLIT_NODE 38 37
ADD_NODE 59
SPLIT_NODE 33 38
ADD_NODE 59
DELETE_NODE 62
ADD_NODE 18
DELETE_SUBTREE 8
ADD_NODE 0
SPLIT_NODE 1 5
ADD_NODE 59
SPLIT_NODE 18 47
ADD_NODE 5
SPLIT_NODE 73 17
ADD_NODE 1
SPLIT_NODE 0 1
ADD_NODE 79
SPLIT_NODE 0 1
ADD_NODE 76
SPLIT_NODE 0 81
ADD_NODE 1
SPLIT_NODE 0 1
ADD_NODE 76
SPLIT_NODE 79 54
ADD_NODE 72
SPLIT_NODE 76 86
DELETE_NODE 47
SPLIT_NODE 1 5
DELETE_SUBTREE 17
SPLIT_NODE 18 75
RIGHTMOST_CHILD 73
RIGHTMOST_CHILD 78
ADD_NODE 5
DELETE_NODE 77
SPLIT_NODE 72 88
DELETE_SUBTREE 89
RIGHTMOST_CHILD 73
RIGHTMOST_CHILD 76
RIGHTMOST_CHILD 92
ADD_NODE 80
DELETE_NODE 87
SPLIT_NODE 59 70
DELETE_SUBTREE 5
SPLIT_NODE 90 84
ADD_NODE 71
SPLIT_NODE 72 88
ADD_NODE 93
DELETE_NODE 88
ADD_NODE 54
DELETE_SUBTREE 84
ADD_NODE 90
RIGHTMOST_CHILD 81
RIGHTMOST_CHILD 99
ADD_NODE 85
DELETE_NODE 59
SPLIT_NODE 0 85
DELETE_SUBTREE 18
SPLIT_NODE 90 78
ADD_NODE 93
SPLIT_NODE 54 100
ADD_NODE 98
SPLIT_NODE 104 101
SPLIT_NODE 104 108
DELETE_SUBTREE 93
ADD_NODE 108
SPLIT_NODE 90 78
RIGHTMOST_CHILD 1
ADD_NODE 103
DELETE_NODE 79
SPLIT_NODE 108 110
DELETE_SUBTREE 85
SPLIT_NODE 104 109
ADD_NODE 95
SPLIT_NODE 104 114
ADD_NODE 110
SPLIT_NODE 90 73
ADD_NODE 90
SPLIT_NODE 118 111
ADD_NODE 116
SPLIT_NODE 104 114
ADD_NODE 110
SPLIT_NODE 104 96
DELETE_NODE 114
SPLIT_NODE 110 123
DELETE_SUBTREE 101
SPLIT_NODE 124 122
RIGHTMOST_CHILD 104
RIGHTMOST_CHILD 90
ADD_NODE 103
DELETE_NODE 74
SPLIT_NODE 110 125
DELETE_SUBTREE 119
RIGHTMOST_CHILD 124
RIGHTMOST_CHILD 104
RIGHTMOST_CHILD 90
ADD_NODE 90
DELETE_NODE 1
SPLIT_NODE 118 120
DELETE_SUBTREE 127
RIGHTMOST_CHILD 70
RIGHTMOST_CHILD 104
RIGHTMOST_CHILD 0
ADD_NODE 108
DELETE_NODE 125
SPLIT_NODE 0 103
DELETE_SUBTREE 116
ADD_NODE 96
RIGHTMOST_CHILD 78
RIGHTMOST_CHILD 104
ADD_NODE 130
DELETE_NODE 95
SPLIT_NODE 118 130
DELETE_SUBTREE 134
RIGHTMOST_CHILD 117
RIGHTMOST_CHILD 129
RIGHTMOST_CHILD 108
ADD_NODE 73
DELETE_NODE 133
SPLIT_NODE 108 131
DELETE_SUBTREE 136
RIGHTMOST_CHILD 118
RIGHTMOST_CHILD 112
RIGHTMOST_CHILD 130
ADD_NODE 110
DELETE_NODE 128
SPLIT_NODE 90 73
DELETE_SUBTREE 113
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 131
RIGHTMOST_CHILD 138
ADD_NODE 117
DELETE_NODE 108
SPLIT_NODE 103 112
DELETE_SUBTREE 78
RIGHTMOST_CHILD 110
RIGHTMOST_CHILD 112
RIGHTMOST_CHILD 103
ADD_NODE 68
DELETE_NODE 90
SPLIT_NODE 124 126
DELETE_SUBTREE 130
RIGHTMOST_CHILD 70
RIGHTMOST_CHILD 124
RIGHTMOST_CHILD 142
ADD_NODE 129
DELETE_NODE 142
SPLIT_NODE 110 138
DELETE_SUBTREE 96
RIGHTMOST_CHILD 139
RIGHTMOST_CHILD 144
RIGHTMOST_CHILD 140
ADD_NODE 0
DELETE_NODE 139
SPLIT_NODE 111 104
DELETE_SUBTREE 143
RIGHTMOST_CHILD 117
RIGHTMOST_CHILD 123
RIGHTMOST_CHILD 103
ADD_NODE 129
DELETE_NODE 138
SPLIT_NODE 104 124
DELETE_SUBTREE 123
RIGHTMOST_CHILD 126
RIGHTMOST_CHILD 135
RIGHTMOST_CHILD 144
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 124
RIGHTMOST_CHILD 117
RIGHTMOST_CHILD 110
RIGHTMOST_CHILD 146
RIGHTMOST_CHILD 104
RIGHTMOST_CHILD 112
