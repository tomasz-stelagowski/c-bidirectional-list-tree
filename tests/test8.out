OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
-1
-1
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
38
-1
-1
OK
OK
OK
OK
16
-1
37
OK
OK
OK
OK
4
-1
-1
OK
OK
OK
OK
-1
44
-1
OK
OK
OK
OK
18
14
42
OK
OK
OK
OK
8
14
-1
OK
OK
OK
OK
18
-1
-1
OK
OK
OK
OK
-1
49
10
OK
OK
OK
OK
OK
OK
17
OK
OK
OK
OK
-1
53
42
OK
OK
OK
OK
-1
-1
28
OK
OK
OK
OK
60
28
44
OK
OK
OK
OK
-1
-1
61
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
77
-1
OK
OK
OK
OK
59
86
-1
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
79
-1
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
90
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
124
119
OK
OK
OK
OK
126
124
118
OK
OK
OK
OK
-1
124
103
OK
OK
OK
OK
OK
-1
124
OK
OK
OK
OK
-1
-1
131
OK
OK
OK
OK
135
-1
-1
OK
OK
OK
OK
132
-1
-1
OK
OK
OK
OK
138
-1
141
OK
OK
OK
OK
-1
143
-1
OK
OK
OK
OK
129
-1
-1
OK
OK
OK
OK
140
-1
141
OK
OK
OK
OK
-1
-1
-1
146
126
140
145
-1
149
-1
